skll/config.py

```diff
@@ -141,7 +143,12 @@ def __init__(self):
 
     def _find_invalid_options(self):
         """
-        Returns the set of invalid options specified by the user
+        Find the set of invalid options specified by the user.
+
+        Returns
+        -------
+        invalid_options : set
+            The set of invalid options specified by the user.
```

Comment:
Do we want to specify the type of the set elements to expect? eg `set of str`. This change would apply to a lot of other functions as well.